ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -88,8 +88,21 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
 ))@
 
 @[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker python3 python3-catkin-pkg-modules python3-pip 
-RUN pip3 install -U auto_abi_checker
+@{common_deps='python3 python3-catkin-pkg-modules python3-pip'
+ubuntu_before_focal=('xenial', 'bionic')
+}@
+@[os_name == 'ubuntu' and os_code_name not in ubuntu_before_focal]@
+# Focal/Groovy abi-compliance-checker package has a bug that breaks python invocation
+# See: https://github.com/lvc/abi-compliance-checker/pull/80#issuecomment-652521014
+# Install 2.3 version from source, needs perl
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y curl make perl
+RUN curl -sL https://github.com/lvc/abi-compliance-checker/archive/2.3.tar.gz | tar xvz -C /tmp && \
+    make install prefix=/usr -C /tmp/abi-compliance-checker-2.3 && \
+    rm -fr /tmp/abi-compliance
+@[else]@
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y abi-compliance-checker @(common_deps)
+@[end if]@
+RUN pip3 install -U auto_abi_checker @(common_deps)
```

Comment:
Doesn't this try to `pip3 install` the Debian package names `python3`, `python3-catkin-pkg-modules`, `python3-pip`?